code/controllers/configuration.dm

```diff
@@ -69,6 +69,7 @@
 	var/forumurl = "http://tgstation13.org/phpBB/index.php" //default forums
 	var/rulesurl = "http://www.tgstation13.org/wiki/Rules" // default rules
 	var/githuburl = "https://www.github.com/tgstation/-tg-station" //default github
+	var/githubrepoid = "3234987"  //default github

```

Comment:
Don't have a default in here, have it in the config. that way somebody can disable the feature by commenting this out.

Giving server ops the ability to control if the game decides to access third party apis or run shell() commands is important enough, and making them have to define the config as a blank string is wasteful.